workflow/rose-suite.conf

```diff
@@ -1,66 +1,72 @@
+# See meta/rose-meta.conf for parameter descriptions
+
 [template variables]
+#--------------------------------------------------
+# Date, Time, Location, Satellites
+#--------------------------------------------------
 START="2019-03-20"
 END="2019-03-21"
-SATELLITES="aqua", "terra"
 LOCATION="beaufort_sea"
 CRS="EPSG:3413"
+# BBOX: "X1,Y1,X2,Y2"
```

Comment:
Should probably note that it's lower left, upper right -- I think we get wonky results if the box is specified upper left lower right instead.

Reply:
Good point! I've updated that here and in the metadata file.